bindings/nodejs/npm/darwin-x64/README.md

```diff
@@ -0,0 +1,3 @@
+# `@opendal/lib-darwin-x64`
```

Comment:
Maybe it's better to remove those meaningless README? I prefer to add a README under `npm/`

Reply:
This can be used for display on the package's npm page, like https://www.npmjs.com/package/@opendal/lib-darwin-x64. It still has some use. 🤣